config/envs/default.py

```diff
@@ -29,6 +29,8 @@ class DefaultConfig(object):
     AUTHENTICATOR_HOST = os.environ.get("AUTHENTICATOR_HOST", "authenticator")
     COOKIE_DOMAIN = os.environ.get("COOKIE_DOMAIN", None)
     SECRET_KEY = os.environ.get("SECRET_KEY", "dev")
+    SUBMIT_DEADLINE = os.environ.get("SUBMIT_DEADLINE", "15/11/2023")
+    RETURNS_PERIOD = os.environ.get("RETURNS_PERIOD", "April to September 2023")
```

Comment:
Returns period won't change so I think we can just make it a constant instead. Submit deadline may not either but this gives us the ability to extend it without a code change should they want to extend it.

Reply:
I moved them here with the thought that they may both have to change for different funds if more of them are onboarded, if that's not a possibility for a while I can move them to constants/ hardcode the returns period :)

Reply:
Yep fair enough